api_spec.yaml

```diff
@@ -3660,11 +3626,15 @@ paths:
 
         The request must originate from public tenant, and the new tenant will use connectionUriDomain and app from which the request originates.
 
+        Note: Updation of core config is not allowed for the default connectionUriDomain, public app and tenant.
+
       operationId: createOrUpdateTenantPUT
       parameters:
```

Comment:
Errors
Hide
 
Semantic error at paths./appid-{appId}/(tenantId}/recipe/userid/map.post.parameters.4.name
Path parameter "tenantId" must have the corresponding {tenantId} segment in the "/appid-{appId}/(tenantId}/recipe/userid/map" path
Jump to line 37
Semantic error at paths./appid-{appId}/(tenantId}/hello
Declared path parameter "appId" needs to be defined within every operation in the path (missing in "put", "post", "delete"), or moved to the path-level parameters object
Jump to line 2989

Reply:
Maybe this is cause you are doing `(tenantId}`

Reply:
Done